### ISSUE TITLE
Ore Textures Visibility

### ISSUE THREAD

**Issue by LuciustheDragon** [TechReborn/TechReborn#3141]
Deepslate Nikolite Ore is extremely difficult to distinguish from regular deepslate even with a reasonable monitor and no visual impairment. The lack of contrast for this ore in particular makes this an accessibility issue. 

The images here compare Nikolite to Modern Industrialization's Deepslate Lignite ore in an unedited screenshot and in monochrome. Both are difficult to see, but the lignite is still visible even in monochrome. The Nikolite is not. 

![2023-06-09_11 30 06](https://github.com/TechReborn/TechReborn/assets/70560762/d07110af-cbfe-498e-a93d-83687d2a1b3b)
![2023-06-09_11 30 06](https://github.com/TechReborn/TechReborn/assets/70560762/30f1ee99-a231-427c-977f-7622d979a604)

If someone could assess the various Tech Reborn ores and update textures as necessary for better accessibility, that would be much appreciated.

**Comment by drcrazy** [comment]
We do not have Nikolite Ore in TR.
